x/staking/keeper/val_state_change.go

```diff
@@ -306,15 +307,19 @@ func (k Keeper) bondValidator(ctx sdk.Context, validator types.Validator) (types
 	// delete the validator by power index, as the key will change
 	k.DeleteValidatorByPowerIndex(ctx, validator)
 
+	// delete from queue if present
+	k.DeleteValidatorQueue(ctx, validator, validator.UnbondingTime, validator.UnbondingHeight)
```

Comment:
Why pass `validator.UnbondingTime` and `validator.UnbondingHeight`? Can't we just retrieve them in `DeleteValidatorQueue` since we pass `validator` anyway?